mtags/src/main/scala/scala/meta/internal/pc/HoverMarkup.scala

```diff
@@ -16,17 +16,14 @@ object HoverMarkup {
       docstring: String
   ): String = {
     val markdown = new StringBuilder()
-    val needsExpressionType = !symbolSignature.endsWith(expressionType)
```

Comment:
Is this change causing the changes to all the tests? Could we instead take an additional parameter that will be set if range is present?

Reply:
good idea, done